public/Sjofn/homework1.html

```diff
@@ -0,0 +1,72 @@
+<html lang="en">
+<head>
+    <title>Homework 1</title>
+    <style>
+        div {
+            background-color: dodgerblue;
+        }
+        #myId{
+            color: tomato;
+            background-color: cadetblue;
```

Comment:
I suggest you to not change background color of a simple button, - a default one looks more natural.